time time_stamp  # Message 발행 시간 (ros::Time)
int32 index       # DI/DO Signal Index Enum
int32 value      # Signal Value